cmd/bedMaxWig/bedMaxWig.go

```diff
@@ -22,7 +22,7 @@ func bedMaxWig(infile string, database string, chromsizeFile string, outfile str
 	var i int
 
 	for i = 0; i < len(sizes); i++ {
-		chromSlice = WigChromToSlice(wigData, sizes[i].Size, sizes[i].Name)
+		chromSlice = wig.ChromToSlice(wigData, sizes[i].Name)
```

Comment:
I think this file was changed to bedValueWig.go back in the spring or early summer. I think there will be a conflict if we try to merge? My guess is this code was written without having done a `git pull` for the most recent version of main. Might need to also check that `go test ...` is also happy.